packages/apollo/package.json

```diff
@@ -54,7 +54,7 @@
     "git-parse": "1.0.3",
     "git-rev-sync": "1.12.0",
     "glob": "7.1.4",
-    "graphql": "~14.2.1",
+    "graphql": "14.0.2-14.2.0 || ^14.3.1",
```

Comment:
```suggestion
    "graphql": "14.0.2 - 14.2.0 || ^14.3.1",
```

Reply:
Appreciate the inline suggestion, but I went ahead and resolved in a commit before I could see them ;P

Resolved in 363a01f